.npmignore

```diff
@@ -1,2 +1,8 @@
 src/
-*.jsx
+test/
+./*.js
+./*.jsx
+./*.json
+.babelrc
+.idea/
+.floo*
```

Comment:
Elsewhere, I've replaced `.npmignore` in favour of a whitelist approach; providing the `files` property inside the `package.json`. i wonder if that might be a better approach.

Reply:
Have a read of https://medium.com/@jdxcode/for-the-love-of-god-dont-use-npmignore-f93c08909d8d and see what you think.

Reply:
Ah yes, brilliant, that's exactly what I wanted to do, thanks @daniel-ac-martin ! Fixed